### PR TITLE
Form link

### PR DESCRIPTION
**Pull request by C0deJack**
Created a SQL link to the CMS form that takes the data from the db (which will be the current data displayed in the portfolio page) and displays it in the form inputs ready to be edited if required

### CODE REVIEW

File: cms.php [hunk 1]
```diff
@@ -1,72 +1,69 @@
...
-    <input type="tel" name="tel">Telephone Number
+<form method="post" action="cms.php">
+    <input type="tel" value="<?php echo $phone; ?>" name="tel">Telephone Number
     <input type="submit" value="update">
 </form><br>
-<form method="post" action="#">
-    <input type="text" name="about">About me paragraph
+<form method="post" action="cms.php">
+    <textarea rows="8" cols="150"  name="about" form="about">
+        <?php echo $about; ?></textarea>
```
Comment (o-tahboub) [inline cms.php:45]: Suggest to move</textarea> to next line fro clarity.

File: cms.php [hunk 2]
```diff
@@ -1,72 +1,69 @@
...
 
-<form method="post" action="#">
-    <h4>Project holder 3</h4><br>
-    <input type="text" name="project-img">Image file name<br>
-    <input type="text" name="project-title">Project Title (max 100 characters)<br>
-    <input type="text" name="project-text">Project Description (max 2000 characters)<br>
-    <input type="text" name="project-link">Project link URL<br>
-    <input type="submit" value="update">
-</form>
+foreach ($result1 as $list) {
```
Comment (o-tahboub) [inline cms.php:52]: Maybe turn this into a function, also look into embedding php in html as it might be easier.

**Review by o-tahboub — APPROVED** [review]

File: cms.php (hunk 1 above)
Comment (C0deJack) [reply cms.php:45]: done

File: cms.php (hunk 2 above)
Comment (C0deJack) [reply cms.php:52]: Yes this is a good idea. I will add this change to the next branch

**Pull request merged**